### DISCUSSION TITLE
Auto pause video on change?

### DISCUSSION THREAD

**Discussion by berianwilliams** [sachinchoolur/lightGallery#1049] [Q&A] [answered]
I'm really liking lightGallery so far.  I've got an inline gallery (constructed via JS) with a combination of images and videos.  When I play a YouTube video it pauses automatically on transition, which I really like.  However, when I use the autoplay option, it will also change slides while a video is playing.  I was wondering whether there was an option to prevent this, and to only change slides when either on an image, or when the video is paused?

**Comment by sachinchoolur** [comment, marked as answer]
Hey @berianwilliams,

Currently, it is not possible. But, it is not hard to implement. I'll try to include it in the next version
